lib/dsl/validator.js

```diff
@@ -112,7 +113,7 @@ const configPropsValidations = {
   },
   PACKAGE_NAME: {
     type: 'qualifiedName',
-    pattern: ALPHABETIC_LOWER,
+    pattern: PACKAGE_NAME_PATTERN,
```

Comment:
This basically means that a package could be like `_a._b._c`

Reply:
Correct.  

@ruddell regex from above has the ability to start with _. 

Then I went to:

https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

> This can occur if the domain name contains a hyphen or other special character, if the package name begins with a digit or other character that is illegal to use as the beginning of a Java name, or if the package name contains a reserved Java keyword, such as "int". In this event, the suggested convention is to add an underscore

Reply:
Technically those work

Reply:
yup they are valid